docker-assets/manageiq.conf

```diff
@@ -67,8 +67,8 @@ RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
   RewriteCond %{REQUEST_URI} ^/ansibleapi
   RewriteCond %{REQUEST_URI} !(.*)/$
   RewriteRule ^(.*)$ $1/ [R,NE,L]
-  ProxyPass /ansibleapi/ http://localhost:54321/api/
-  ProxyPassReverse /ansibleapi/ http://localhost:54321/api/
+  ProxyPass /ansibleapi/ http://#{ANSIBLE_SERVICE_NAME}/api/
```

Comment:
We only had to expose to the outside world because the other appliances needed to talk to it via the external network.  Inside of a kube cluster I would expect they would be in the same project and thus it can just hit the service directly.